shuup/admin/toolbar.py

```diff
@@ -184,6 +189,11 @@ def for_model(cls, model, **kwargs):
             except NoModelUrl:
                 return None
             kwargs["url"] = url
+
+        if 'required_permissions' not in kwargs:
+            url = urlparse(kwargs['url']).path
+            kwargs['required_permissions'] = (resolve(url).url_name,)
```

Comment:
how this solves the issue? the best permission to add this model is:
`"%s.add_%s" % (model._meta.app_label, model._meta.model_name)`.. makes sense?
That way it will be `shuup.add_product` for Product model. This is the way we do for admin modules.